html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="GET">
+    <label for="email">Email</label>
+    <div id="email">
```

Comment:
This `id` should be on the `input` to match with the `label`

Reply:
Okay what about for the radio inputs where there are multiple options?  Do we just put the id on the first input?